iXBRLViewerPlugin/viewer/src/i18n/en/currencies.json

```diff
@@ -12,10 +12,15 @@
     "centsGBP": "pence",
     "accuracy3": "thousandths",
     "accuracy2": "hundredths",
+    "accuracy1": "tenths",
```

Comment:
pre-existing: Naming scheme doesn't seem very intuitive
pre-existing: The inclusion of accuracy in the currencies file doesn't seem very intuitive

Nit: How do we handle decimals?

Reply:
> pre-existing: Naming scheme doesn't seem very intuitive
> pre-existing: The inclusion of accuracy in the currencies file doesn't seem very intuitive

They're used for both scale and accuracy, but I think `scale3` is a more intuitive reference to `thousands` than `accuracy-3`. Maybe making that change and moving them all to a `scale.json` addresses both of these concerns?

> Nit: How do we handle decimals?

Not exactly sure what you mean, but `accuracy1..3` represent 1-to-3 decimal places, respectively. Scale is defined [in spec](https://www.xbrl.org/specification/inlinexbrl-part1/rec-2013-11-18/inlinexbrl-part1-rec-2013-11-18.html#sec-nonFractions) as an integer representing a power of 10, so we don't ever need to worry about `accuracy2.5`, if that's what you're asking about.

Reply:
Will defer to @brettkail-wk @paulwarren-wk here for their xbrl expertise, does ^ seem reasonable?

Reply:
I agree that "scale" is better than "accuracy".  I was initially confused that `-` was meaning "minus" rather than just being a separator i.e. `accuracy-3` was actually "accuracy = -3", but I don't have a suggestion for how to improve that aspect.

Reply:
See latest commits, ended up creating "scale" object with "-3", "1", "9", etc. as keys.